src/utils/index.ts

```diff
@@ -1 +1,17 @@
 export * from './api';
+
+export const getTodayDate = (): string => {
+  const today = new Date();
+  const dd = today.getDate();
+  const mm = today.getMonth() + 1;
+  const year = today.getFullYear();
+
+  const day = dd < 10 ? `0${dd}` : dd;
+  const month = mm < 10 ? `0${mm}` : mm;
+
+  return `${year}-${month}-${day}`;
+};
```

Comment:
nice solution. To make story short: I want you to refactor it with `luxon` library there.